fuse_models/src/imu_2d.cpp

```diff
@@ -269,4 +211,71 @@ void Imu2D::process(const sensor_msgs::Imu::ConstPtr& msg)
   sendTransaction(transaction);
 }
 
+void Imu2D::processDifferential(const geometry_msgs::PoseWithCovarianceStamped& pose,
+                                const geometry_msgs::TwistWithCovarianceStamped& twist, const bool validate,
+                                fuse_core::Transaction& transaction)
+{
+  auto transformed_pose = std::make_unique<geometry_msgs::PoseWithCovarianceStamped>();
+  transformed_pose->header.frame_id =
+      params_.orientation_target_frame.empty() ? pose.header.frame_id : params_.orientation_target_frame;
+
+  if (!common::transformMessage(tf_buffer_, pose, *transformed_pose))
+  {
+    ROS_ERROR_STREAM("Cannot transform pose message with stamp "
```

Comment:
Should these be throttled? On one hand, they should not happen. On the other hand, if tf is misconfigured/having issues it tends to be very chatty.

Reply:
Fair enough. Changed so it matches the logging message that already is in `common::transformMessage()`:
https://github.com/locusrobotics/fuse/blob/9119624be8f0837a6d43c3b70b7c33d118d3a6da/fuse_models/include/fuse_models/common/sensor_proc.h#L234-L235
That is, `5s` throttling and `WARN` level.

Maybe we could simply remove the logging message from here, but then I believe it'd be hard to figure out which sensor model generated it. Just thinking out loud.

Reply:
@ayrton04 Does it look good for you like this?

Reply:
LGTM!